lib/utils/filter.rb

```diff
@@ -175,7 +175,15 @@ def field?(proposed_field)
       # Currently we only know about a field if it is present in a at least one row of the raw data.
       # If we have no rows in the raw data, assume all fields are acceptable (and rely on failing to match on value, nil)
       return true if raw_data.empty?
-      list_fields.include?(proposed_field) || is_field_lazy?(proposed_field)
+
+      # Most resources have Symbol keys in their raw data.  Some have Strings (looking at you, `shadow`).
```

Comment:
I also noticed that if you create your filter table data via "JSON.parse(some_api_response)" you'll get string keys.
I was using ` @data.map! { |entry| Hash[entry.map { |k,v| [k.to_sym, v] }] }` to get the keys to symbols.